src/processing/tegnespillet/README.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Lærerveiledning - Tegneprogram
+level: 1
+language: nb
+---
+
+# Om oppgaven {.activity}
+I dette prosjektet lager vi et tegneprogram slik at man etterpå kan lage sin egen kunst. Man kan tegne med forskjellige farger, og lage figurer som ligner på blant annet denne:
+
+![](gronttre.png)
+
+## Oppgaven passer til: {.check}
+ __Fag__: Kunst og håndtverk, matematikk, programmering.
+__Anbefalte trinn__: 8. trinn til VG3.
+__Tema__: Tegning, koordinatsystem, brukerinteraksjon.
+__Tidsbruk__: Dobbelttime eller mer.
+
+## Kompetansemål {.challenge}
+- [ ] __Kunst og håndtverk, 4. trinn__: bruke enkle funksjoner i digitale bildebehandlingsprogram
+- [ ] __Matematikk, 4. trinn__: lese av, plassere og beskrive posisjoner i rutenett, på kart og i koordinatsystemer, både med og uten digitale verktøy
+- [ ] __Programmering, 10. trinn__: bruke flere programmeringsspråk der minst ett er tekstbasert
+- [ ] __Programmering, 10. trinn__: bruke grunnleggende prinsipper i programmering, slik som løkker, tester, variabler, funksjoner og enkel brukerinteraksjon
+
+## Forslag til læringsmål {.challenge}
+- [ ] Elevene kan lage og bruke enkle funksjoner for bildebehandling.
+- [ ] Elevene kan bruke kode til å lese av posisjonen til musepekeren i et koordinatsystem, og bruke denne informasjonen videre.
+- [ ] Elevene kan bruke kode til å endre funksjoner og verktøy etter brukerinteraksjon.
+
+## Forslag til vurderingskriterier {.challenge}
+Det er mange ulike måter en kan vurdere et programmeringsprosjekt, og her må en selv vurdere hva som er den beste måten ut ifra hvilket fag man jobber i, hvilken aldergruppe og hvilet nivå elevene er på, hva man ønsker å teste og hvor mye tid man har til rådighet til å jobbe med prosjektet. I vårt lærerdokument har vi blant annet beskrevet ulike måter dette kan gjøres på, i tillegg til en del andre nyttige tips til hvordan man underviser i programmering.
```

Comment:
Ja, men dei ligg ikkje ute, og eg ville ikkje lage ei lenke der før eg visste kor hen dokumentet kom til å ligge.
Og så gjeld det alle rettleiingane me har laga. Så det bør bli skrive eit skript for å endre alle til den teksten _med hyperlenke_.